2024/calcite-react/demo/src/main.tsx

```diff
@@ -16,12 +16,17 @@ import {
   panelHeading,
 } from './config/application.json';
 
-// Step 4: Calcite Components Assets (Recommended to use CDN based on documentation)
-// Please note: You can also manage assets locally by copying assets directly from Calcite Components in your project.
-import { setAssetPath } from '@esri/calcite-components/dist/components/index';
-setAssetPath(
-  'https://cdn.jsdelivr.net/npm/@esri/calcite-components/dist/calcite/assets',
-);
+// Step 4: Calcite Components
+// 4a. Use setAssetPath to the path where assets were copied (occurs in package.json)
+import { setAssetPath } from '@esri/calcite-components/dist/components';
+const { href } = new URL(`${window.location.href}assets`);
```

Comment:
while it works here, href might not include a trailing slash (if app is deployed on non-root path), so this would break - do we have to handle that case here or is that too complex for a small demo app?

also, if we make a (potentially unsafe) assumption that the app is deployed at the root only, then the above could be made a bit simpler:

```suggestion
const { href } = new URL('assets', window.location.href);
```

Reply:
No longer needed since we'll be using `defineCustomElements` instead.